sdk/spring/azure-spring-boot/pom.xml

```diff
@@ -149,12 +149,10 @@
       <optional>true</optional>
     </dependency>
 
-    <!--Qpid-->
     <dependency>
-      <groupId>org.apache.qpid</groupId>
-      <artifactId>qpid-jms-client</artifactId>
-      <version>0.43.0</version> <!-- {x-version-update;org.apache.qpid:qpid-jms-client;external_dependency} -->
-      <optional>true</optional>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-servicebus-jms</artifactId>
+      <version>0.0.2</version> <!-- {x-version-update;com.microsoft.azure:azure-servicebus-jms;external_dependency} -->
```

Comment:
Should we add make this dependency optional here？

Reply:
Updated.